composer.json

```diff
@@ -62,7 +62,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.0.0"
+            "php": "5.6.0"
```

Comment:
I think we need to change manual (replace section about installing php7 with section about installing php 5.6).